static/js/impala/stats/stats.js

```diff
@@ -2,7 +2,7 @@
   'use strict';
 
   // `$` is passed by jQuery itself when calling `jQuery(stats_stats)`.
-  var stats_stats = function ($, injectedSessionStorage) {
+  const stats_stats = function ($, injectedSessionStorage) {
```

Comment:
😱 `const`!

Reply:
When's the PR coming to convert everything to `const`? 😉

Reply:
https://github.com/mozilla/addons-server/pull/15131

![](https://media3.giphy.com/media/UTYz3M8lcTvqaVbSo9/giphy.gif)